specs/integration/1 - SearchFeature.js

```diff
@@ -0,0 +1,33 @@
+/*
+As an anonymous user, I want to perform various search queries from the homepage
+in order to find datasets, orgs, reuses and users.
```

Comment:
Out of curiosity, why not store this BDD-style description as a comment rather than as the `description` itself?